tango_simlib/tango_sim_generator.py

```diff
@@ -188,7 +189,7 @@ def attr(tango_device_instance, new_val):
 
     # Sim test interface static attribute `attribute_name` info
     # Pick the first model instance in the dict.
-    controllable_attribute_names = models.values()[0].sim_quantities.keys()
+    controllable_attribute_names = list(itervalues(models))[0].sim_quantities.keys()
```

Comment:
IMHO this is complicating the not so complicated, how would you handle an exception.
This line of code would be intimidating for me to read if I was a noob.

Perhaps, break it down.

Reply:
I have changed it to multiple lines but what exceptions would you like me to handle?
Could that be done in a separate PR as `adding exception handling` seems to be outside of the scope of py2 to py3 conversion.

Reply:
There are other alternatives. I opted for option 4 as it is the most explicit see https://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list 

I could also bring in `dict` from `future.utils`  and just do a `list` casting.